spring-cloud-dataflow-server/docker-compose-influxdb.yml

```diff
@@ -25,7 +25,7 @@ services:
       - '8086:8086'
 
   grafana:
-    image: springcloud/spring-cloud-dataflow-grafana-influxdb:${DATAFLOW_VERSION:?DATAFLOW_VERSION is not set!}
```

Comment:
The `latest` is not a valid image tag nether for the SCDF or Skipper server nor for the Stream or Task apps.  An explicit app version tags is required.

Reply:
So this is not always latest, you can supply the version either as a variable `${DATAFLOW_VERSION}` or if nothing is specified then the `latest` is chosen for local quick testing.